oaknational/Oak-Web-Application
# Feat/1043 same content on teacher hompepage as homepage

## Description

- Refactor home page into shared components
- Adds tests for teacher page
- Put content from homepage on teachers homepage
- Makes link of acorn icon go to  `beta/teachers` homepage not the homepage.

## Issue(s)

Fixes #1043 

## How to test

WARNING - this touches publicly available home page as well - so please QA thoroughly.

1. Go to [homepage](https://oak-web-application-2dt8w9wrc-oak-national-academy.vercel.app/)
2. Check homesite has no new bugs and is displaying correctly
4. Got [teachers homepage](https://oak-web-application-2dt8w9wrc-oak-national-academy.vercel.app/beta/teachers)
5. Check content is displayed correctly - links and. forms work
6. Acorn icon should take you to this page

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
